From 7f4d9950861463e94116dc08410ccc48c69121e9 Mon Sep 17 00:00:00 2001
From: Noam Camus <ncamus@habana.ai>
Date: Wed, 08 Aug 2018 17:49:58 +0300
Subject: [PATCH] BTL: update preboot load address

It is moved from SRAM offset 4MB to 0MB
In order to support ONIX that have only 2MB of SRAM
---

diff --git a/board/habanalabs/dali/btl/Makefile b/board/habanalabs/dali/btl/Makefile
index 96e59d3..807429b 100644
--- a/board/habanalabs/dali/btl/Makefile
+++ b/board/habanalabs/dali/btl/Makefile
@@ -4,9 +4,9 @@
 PCIE_ASMFILE ?= pcie.S
 
 COMMON_CPP_FLAGS  = -DPREBOOT_FILE=../dali-preboot.bin
-COMMON_CPP_FLAGS += -DPREBOOT_ADDR=0x7FF0400000
+COMMON_CPP_FLAGS += -DPREBOOT_ADDR=0x7FF0000000
 COMMON_CPP_FLAGS += -DPREBOOT_ADDR_H=0x7F
-COMMON_CPP_FLAGS += -DPREBOOT_ADDR_L=0xF0400000
+COMMON_CPP_FLAGS += -DPREBOOT_ADDR_L=0xF0000000
 
 COMMON_CPP_FLAGS += -I$(dir $(BR2_EXTERNAL))/specs/dali/asic_reg
 
